src/main/java/uk/gov/hmcts/divorce/divorcecase/model/GeneralReferral.java

```diff
@@ -24,51 +26,59 @@ public class GeneralReferral {
     @CCD(
         label = "Reason for referral?",
         typeOverride = FixedRadioList,
-        typeParameterOverride = "GeneralReferralReason"
+        typeParameterOverride = "GeneralReferralReason",
+        access = {CaseworkerAccessOnlyAccess.class}
```

Comment:
If CaseworkerAccessOnlyAccess is given to every field then can we not add it to the top where GeneralReferral is defined ?